### DISCUSSION TITLE
Promo block page: EDI question

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1370] [NCIDS Launch]
Submitted by Anna during her doc site review:

On Promo block page (https://designsystem-dev.cancer.gov/develop/components/promo-block), is anyone else concerned about using the EDI content in the promo block when we’re using placeholder text to refer to “light” and “dark” modifiers?

**Comment by blilianyu** [comment]
We can update the the promo block image and content to be Annual Plan to be consistent with the promo blocks with no image examples.

Update all content on Promo Block page to be "NCI Fiscal Year 2025 Professional Judgment Budget Proposal" instead of "Annual Plan".

cc: @monika-jaeger

**Reply by monika-jaeger** [reply]
Added all Promo Block examples with "NCI Fiscal Year 2025 Professional Judgment Budget Proposal" [linked here](https://www.figma.com/file/8qu1psEjnehVDoZyWrH7ny/NCIDS-For-All?type=design&node-id=596%3A4041&mode=design&t=xW5MDJS4hgEDajLI-1) for team discussion.

cc: @blilianyu

**Reply by blilianyu** [reply]
Will have to be a dev ticket

**Comment by laurelthrash** [comment]
Notified Anna and Alex on 04/24 via email of this intended change. @andyvanavery31 @sarinapadilla are you available to please create a CR for this issue and add to the Redesign board? Thank you!

**Reply by blilianyu** [reply]
@laurelthrash @andyvanavery31 @sarinapadilla CR has been created #1672
